tensorflow/lite/micro/tools/make/ext_libs/cmsis_nn.inc

```diff
@@ -64,6 +52,6 @@ THIRD_PARTY_CC_HDRS += \
 # code without any modifications.
 INCLUDES += \
   -I$(CMSIS_PATH) \
+  -I$(CMSIS_NN_PATH) \
```

Comment:
let's remove this and instead add `-I(DOWNLOADS_DIR)` to [this list](https://github.com/tensorflow/tflite-micro/blob/bea0931154d8ec38afc06a64ae1244c38a6f848f/tensorflow/lite/micro/tools/make/Makefile#L81-L87).

That will allow us to have the include paths be `#include "cmsis_nn/Include/..."` which would preferable.

Reply:
I prefer to keep it like before. If using CMSIS_NN_PATH option, i.e. an external repo the path may not be cmsis_nn/Include. Ok?

Reply:
Since in TFLM we provide an [option](https://github.com/tensorflow/tflite-micro/pull/1468/files#diff-11d69e56729e5b7cdd1845a3d72024600ac4323334f49d5dc55028273fc7c859R28) to a user to use their own version of CMSIS-NN, having 'cmsis_nn/' would force them to use a downloaded folder name that is set by the glue file. That might not be possible for users. 

  For that, I would prefer use of 'Include/*.h' as well. 
   I understand that using 'cmsis_nn/Include' might be to avoid ambiguities, but since the NN folder from the existing 'CMSIS/NN' will be removed, there shouldn't be any conflict of the NN includes between the existing CMSIS_5 repository and the new CMSIS-NN repository.

Reply:
Chatted with @mansnils and conclusion was that we'll keep as is. It might mean some changes to the Arduino sync but it should not be too much work.